ENIGMAsystem/SHELL/Widget_Systems/TinyFileDialogs/dialogs.cpp

```diff
@@ -0,0 +1,469 @@
+#include "dialogs.h"
+#include "Widget_Systems/widgets_mandatory.h"
+#include <shared/tinyfiledialogs/tinyfiledialogs.h>
+#include <sys/stat.h>
+#include <limits.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <string.h>
+#include <algorithm>
+#include <sstream>
+#include <vector>
+#include <string>
+
+using std::string;
+
+namespace enigma
+{
+    bool widget_system_initialize()
+    {
+        return true;
+    }
+}
+
+bool is_number(string input)
```

Comment:
This probably exists elsewhere in the engine. I don't think you need this, though.